### DISCUSSION TITLE
Which API are you using for calendars?

### DISCUSSION THREAD

**Discussion by daya0576** [daya0576/beaverhabits#39] [Q&A]
Which API are you using for calendars? -- from reddit

**Comment by daya0576** [comment]
For the habit detail page:

- The monthly view calendar uses the [Quasar QDate component](https://quasar.dev/vue-components/date/)
- The yearly view streaks are purely hand-making, consisting of tons of [checkboxes](https://nicegui.io/documentation/checkbox) XD

![image](https://github.com/user-attachments/assets/7a88f69f-dad8-4a78-9b57-d7c0324cc577)

**Reply by ocean-haiyang** [reply]
Caldav
